src/pages/tenders.tsx

```diff
@@ -106,11 +106,7 @@ export const getStaticProps: GetStaticProps = async () => {
   }
 };
 
-const MyTendersPage: FC<IProps> = ({
-  tenders,
-  statusCode = null,
-  host = '',
-}) => {
+const Profile: FC<IProps> = ({ tenders, statusCode = null, host = '' }) => {
```

Comment:
Why have you changed this file?

Reply:
I think it was an accident.. I will fix it later